scripts/dependency/check.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
```

Comment:
This script should be part of `azdev verify` because this script will not run on Windows.

Reply:
Also, like we discussed in scrum, this should execute along with Derek's dependency checks.